test/070-theme-translations.test.js

```diff
@@ -2,6 +2,10 @@ const should = require('should'); // eslint-disable-line no-unused-vars
 const utils = require('./utils');
 const thisCheck = require('../lib/checks/070-theme-translations');
 
+const themePath = require('./utils').themePath;
```

Comment:
We already require this module in a few lines above this - not a big deal but probs cleaner to just use `utils.themePath`

Reply:
good catch, thanks!